packages/vscode-extension/src/commonlib/telemetry.ts

```diff
@@ -52,7 +53,7 @@ export class VSCodeTelemetryReporter extends vscode.Disposable implements Teleme
   }
 
   addSharedProperty(name: string, value: string): void {
-    this.sharedProperties[name] = value;
+    this.sharedProperties[name] = value ?? "";
```

Comment:
The parameter `value` can be changed to optional? Like `value?: string`

Reply:
Fixed, thx!